image/entrypoints/deploy-plan.sh

```diff
@@ -4,6 +4,7 @@ source /usr/local/actions.sh
 
 debug
 setup
+init
```

Comment:
There's some magic somewhere else that makes `init` do a `terraform init` ?

Reply:
yea, its a function in that `action.sh` file

Reply:
```function init() {
  rm -rf "$TF_DATA_DIR"
  (cd "$INPUT_PATH" && terraform init -input=false -backend=false)
}
```